ckanext/nextgeoss/public/nextgeoss.css

```diff
@@ -411,7 +411,7 @@ h4.title a {
   box-shadow: 0px 3px 13px -2px rgb(173, 173, 173);
 }
 
-.fa-search:before {
+.icon-search:before {
```

Comment:
@kmbn this needs to be fa-search instead of icon-search. Because CKAN-2.7.2 uses Font Awesome 3, where icon is replaced with fa